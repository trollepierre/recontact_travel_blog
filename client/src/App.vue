<template>
  <div id="app">
    <app-header/>
    <subscribe-modal/>
    <feedback-modal/>
    <router-view/>
  </div>
</template>

<script>
  import AppHeader from '@/components/AppHeader';
  import FeedbackModal from '@/components/FeedbackModal';
  import SubscribeModal from '@/components/SubscribeModal';

  export default {
    name: 'App',

    components: {
      'app-header': AppHeader,
      'feedback-modal': FeedbackModal,
      'subscribe-modal': SubscribeModal,
    },
  };
</script>

<style>
  body {
    background: aliceblue;
    padding: 0;
    margin: 0;
  }

  #app {
    font-family: "Roboto", "Helvetica Neue", Arial, sans-serif;
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
    text-align: center;
    color: #2c3e50;
  }

  .v--modal {
    border-radius: 5px;
  }

  @media only screen and (min-width: 640px) {
    .page__container {
      max-width: 544px;
    }
  }

  @media only screen and (min-width: 992px) {
    .page__container {
      max-width: 816px;
    }
  }

  @media only screen and (min-width: 1200px) {
    .page__container {
      max-width: 1088px;
    }
  }
</style>
