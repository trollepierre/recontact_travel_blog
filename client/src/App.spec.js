import App from './App.vue'

describe('Component | App', () => {
  let localVue
  beforeEach(() => {
    localVue = createLocalVue()
  })

  describe('template', () => {
    it('should match snapshot', () => {
      // When
      const wrapper = shallowMount(App, { localVue })

      // Then
      expect(wrapper.element).toMatchSnapshot()
    })

    xit('should have AppHeader', () => {
      // When
      const wrapper = shallowMount(App, {
        localVue,
      })

      // Then
      expect(wrapper.find(AppHeader).exists()).toBe(true)
    })
  })
})
