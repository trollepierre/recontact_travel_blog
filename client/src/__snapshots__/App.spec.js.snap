// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | App template should match snapshot 1`] = `
<div
  id="app"
>
  <appheader-stub />
   
  <subscribemodal-stub />
   
  <feedbackmodal-stub />
   
  <vuecomponent-stub />
</div>
`;
